bibtex-actions.el

```diff
@@ -242,12 +242,12 @@ a list of entries, from which they can narrow against a string
 which includes title, author, etc., and then select one.  This
 function will then return the key 'key', resulting in '@key' at
 point."
-  (when (looking-back "@[a-zA-Z0-9]*" 5))
+  (when (eq ?@ (char-before))
```

Comment:
@minad - can you please help me with this?

I think this is actually correct, as in it appears to activate (I get a message that confirms the bibliography is parsed), but it returns no results.

I basically want a user to be able to type only `@`, and have the full list of candidates presented by `corfu`, as they are in `vertico` or `selectrum`, and then to narrow from there.

I can revert the change, but that just doesn't seem right from a UX POV.

Perhaps there's something else I need to add here to do that? Maybe `:predicate`?

Reply:
Maybe @ is part of the input string, such that this will filter out the candidates?

Reply:
> Maybe @ is part of the input prefix (as determined by the start end position of the completion), such that this will filter out the candidates?

Does it make sense to you that the previous version worked as expected?

Reply:
> And if yes, what is the appropriate way to set the positions in that case? Set both begin and end to point?

This works @minad; thanks for the hint!

Reply:
If in the end, we need to support a few variations on this, do you recommend separate functions for each, or one capf?

1. pandoc markdown; syntax is `[@key]` or even the bare `@key`; so rather loose
1. this PR is currently for the forthcoming org-cite, to be merged in org-mode "real soon"; syntax is similar to 1, but tighter `[cite:@key]`.
3. latex; lower priority for me personally, and the syntax is too chaotic to easily constrain